pkgs/development/tools/misc/nimlsp/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, fetchFromGitHub, nim, termbox, pcre }:
+
+let
+  astpatternmatching = fetchFromGitHub {
```

Comment:
I am not to familiar with nim but should this be an extra package or is this not possible?

Reply:
This is awkward but I but fine until we get something like what is done for Go and Rust.